float32[3] source_position
float32[3] source_direction
float32[2] source_aperture
float32 mAs
float32[3] shield_Euler_angles
float32[3] shield_translation
float32[3] shield_size
float32 shield_attCoef
string energy_spectrum_file
string timestamp
